main.tf

```diff
@@ -9,7 +9,7 @@ data "aws_iam_policy_document" "default" {
 }
 
 module "s3_user" {
-  source        = "git::https://github.com/cloudposse/terraform-aws-iam-system-user.git?ref=tags/0.14.0"
+  source        = "git::https://github.com/cloudposse/terraform-aws-iam-system-user.git?ref=tags/0.16.0"
```

Comment:
this is speculative based on the latest version of the module and https://github.com/cloudposse/terraform-aws-iam-system-user/pull/33